opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -1390,11 +1390,14 @@ namespace Opm {
                 const double cellVolume = ebosSimulator_.model().dofTotalVolume(cellIdx);
                 auto& cellRes = ebosResid[ cellIdx ];
 
+                const auto& intQuants = *(ebosSimulator_.model().cachedIntensiveQuantities(cellIdx, /*timeIdx=*/0));
+                unsigned pvtRegionIdx = intQuants.pvtRegionIndex();
```

Comment:
not a big deal: the problem is more low-level than the intensive quantities because that's where the intensive quantities ultimately get their data from.